jobserver/migrations/0045_use_django_json_encode_for_release_requested_files.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.7 on 2022-08-23 15:49
+
+import django.core.serializers.json
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("jobserver", "0044_add_review_fields_to_release_and_releasefile"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="release",
+            name="requested_files",
+            field=models.JSONField(
+                encoder=django.core.serializers.json.DjangoJSONEncoder
```

Comment:
Huh, presumably this is a migration because it needs to load/dump all the current json stored in the db?

Reply:
This is because the model definition changed.  Django migrations track all changes to models, a subset of which change the db state, so that they can auto-generate subsequent migrations.